apps/frontend/src/context/MockContext.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+declare global {
+  interface Window {
+    setMock: (b?: boolean) => void;
+    mockHr: number | null;
+    mockWatt: number | null;
+  }
+}
+
+const mockingOnFromEnvironment =
+  import.meta.env.VITE_USE_MOCK_DATA === 'true' ||
+  import.meta.env.VITE_USE_MOCK_DATA === 'TRUE' ||
+  false;
```

Comment:
Keeps the possibility to toggle on mocking from env-vars, so that we don't have to use this as much locally, and just use `yarn mock`